requirements.txt

```diff
@@ -14,8 +14,12 @@ python-dateutil==2.8.2
 
 vine==1.3.0
 amqp==2.6.1
-kombu>=4.6.11,<5
-celery>=4.4.7,<5.0.0
+kombu>=5.3.0b2,<6.0
```

Comment:
We might as well just pin kombu to the exact version 5.3.0b2 for now, as 6.0 isn't out yet.